lib/qore_date_private.cpp

```diff
@@ -279,6 +331,12 @@ void qore_absolute_time::set(const char* str, const AbstractQoreZoneInfo* n_zone
       return;
    }
 
+   if (xsink && second > 59) {
```

Comment:
Currently Qore doesn't support leap seconds.  according to what I found, there are only conventions regarding when a leap second is added (normally the end of the year or the end of June) but no rules.  Additionally, there is no long-term schedule for leap seconds, rather they are decided relatively shortly in advance on an ad-hoc basis based on the actual difference in the earth's rotation and atomic time.

So we can support leap seconds by changing the `59` to a `60` here, but in my opinion, it's not possible to verify them (at least not for future dates).